packages/synapse-react-client/src/components/OrientationBanner/OrientationBanner.tsx

```diff
@@ -82,7 +82,11 @@ function OrientationBanner(props: OrientationBannerProps) {
         }
         setShowBanner(false)
       }}
-      sx={{ ...defaultSx, ...sx }}
+      sx={{
+        ...defaultSx,
+        ...sx,
+        display: { xs: 'none', md: 'unset' },
```

Comment:
Sorry, I didn't realize that we already had an `sx` object -- can you move this `display` property to the `defaultSx` object?

Reply:
oh yep! that makes more sense. On it!